### ISSUE TITLE
second right scroll bar after renew token

### ISSUE THREAD

**Issue by ahadjiGladiator** [AzureAD/azure-activedirectory-library-for-js#562]
I understand that ADAL renew token every hour using hidden iframe.
However whenever this happens it caused second vertical scroll bar.
I am using bootstrap with fixed top and left menu. if I refresh the page (via browser refresh) the second one will disappear. why this is happening?

**Comment by rohitnarula7176** [comment]
@ahadjiGladiator Can you please explain what do you mean by second scroll bar. I will need to know your application set up to investigate this further.

**Comment by ahadjiGladiator** [comment]
I am using angular 1.6 with bootstrap. my page layout has a fix top and left navigation. Below is the code.
My views will be loaded in "<div ng-view></div>".
obviously if page is longer than browser window, I will have right scroll bar. (vertical)
when I use the scroll bar top nav and left nave will stay and content will scroll. (ng-view)
now after 1 hour when access token needs to renew (behind the scene) when I go to another page (like click on a link) a second (vertical) scroll bar will appear in right side.
it is very similar to if you have an ifram inside html and the content is long.
I think this has something to to with adal hidden ifram when renews the access token.   

`<body data-ng-controller="indexCtrl">
    <div class="navbar navbar-default navbar-fixed-top">
        <div class="navbar-header">
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            <a class="navbar-brand active" href="#/"> Admin</a>
        </div>
        <div class="collapse navbar-collapse">
            <ul class="nav navbar-nav">
            </ul>
            <ul class="nav navbar-nav navbar-right" ng-if="userInfo.isAuthenticated">
                <li class="userinfo"><a href="#/">Welcome {{userInfo.profile.given_name}}<i class="fa fa-user fa-fw"></i></a></li>
                <li><a ng-href="#/" data-ng-click="logout()">Logout<i class="fa fa-sign-out fa-fw pull-right"></i></a></li>
            </ul>
        </div>
    </div>
    <div class="row-offcanvas row-offcanvas-left">
        <div id="sidebar" class="sidebar-offcanvas">
            <div class="col-md-12">
                <h3></h3>
                <ul class="nav nav-pills nav-stacked">
                    <li class="nav-divider"></li>
                    <li class=""><a ng-href="#/">Home<i class="fa fa-home fa-fw pull-right"></i></a></li>
                    <li ng-show="userInfo.isAuthenticated"><a ng-href="#/apilog/">Api Log<i class="fa fa-book fa-fw pull-right"></i></a></li>
                    <li><a ng-href="#/" ng-hide="userInfo.isAuthenticated" ng-click="login()">Login<i class="fa fa-sign-in fa-fw pull-right"></i></a></li>
                </ul>
            </div>
        </div>
        <div id="main">
            <div class="col-md-12">
                <div ng-view></div>
            </div>
        </div>
    </div>
</body>`

**Comment by rohitnarula7176** [comment]
@ahadjiGladiator Can you attach the demo code entirely so I can repro the issue. I will need that to investigate this further.

**Comment by ahadjiGladiator** [comment]
@rohitnarula7176 
Please see complete demo code in the link below:
https://github.com/ahadjiGladiator/SampleCodes

tenant and clientId need to be set in app.js
I have changed line 324 of adal-1.0.14.js  in order to renew token every 20 seconds for testing.
To reproduce: click on "Log" from left menu, the page is long so you will have right scroll bar. wait 20 seconds and click on "Log" again, you will see another right scroll bar will appear under top menu.

**Comment by LastTribunal** [comment]
I confirm this issue.  The iframe will force a scroll when it is rendered, every time!
This is how you fix it: in adal.js find "style="display:none" and replace it with "style="display:block"

**Comment by MatthiasDebaere** [comment]
I also noticed this second scrollbar in our application, and I believe it's because the user agent stylesheet (a browser's default style) added a border to all iframe elements. Since we had set the height value of the parent <div> element in the <body> to a 100%, the border of the iframe added 4px to the window, which resulted in the extra scrollable area.

If I set `border: none` on iframe elements in my css file, it renders no border anymore and tadaa, no more unwanted scrollbar! :)
